deepchecks/vision/README.rst

```diff
@@ -93,14 +93,11 @@ Using conda
 
 Check out the following tutorials for a quick start with deepchecks for CV:
 
-- `Deepchecks Example - Simple Image Classification Tutorial <https://docs.deepchecks.com/en/stable/tutorials/vision/examples/plot_simple_classification_tutorial.html\
-  ?utm_source=github.com&utm_medium=referral&utm_campaign=readme&utm_content=try_it_out>`_
+- `Deepchecks for Simple Image Classification Tutorial (for data without model) <https://docs.deepchecks.com/stable/user-guide/vision/auto_tutorials/plot_simple_classification_tutorial.html>`_
 
-- `Deepchecks for Object Detection Tutorial <https://docs.deepchecks.com/en/stable/tutorials/vision/examples/plot_detection_tutorial.html \
-  ?utm_source=github.com&utm_medium=referral&utm_campaign=readme&utm_content=try_it_out>`_
+- `Deepchecks for Object Detection Tutorial <https://docs.deepchecks.com/stable/user-guide/vision/auto_tutorials/plot_detection_tutorial.html>`_
 
-- `Deepchecks for Classification Tutorial <https://docs.deepchecks.com/en/stable/tutorials/vision/examples/plot_classification_tutorial.html \
-  ?utm_source=github.com&utm_medium=referral&utm_campaign=readme&utm_content=try_it_out>`_
+- `Deepchecks for Classification Tutorial <https://docs.deepchecks.com/stable/user-guide/vision/auto_tutorials/plot_classification_tutorial.html>`_

```

Comment:
The UTM parameters are important for the validation of the readme quality. Can you keep them?

Reply:
Sure